garrysmod/lua/postprocess/dof.lua

```diff
@@ -13,12 +13,16 @@ local NUM_DOF_NODES = 16
 
 function DOF_Kill()
 
-	for k, v in pairs( DOF_Ents ) do
+	for i = #DOF_Ents, 1, -1 do
 
-		if ( IsValid( v ) ) then
+		local v = DOF_Ents[i]
+
+		if ( v:IsValid() ) then

```

Comment:
I believe this change is safe - it's only entities in here, and even `NULL`s have `:IsValid`

Reply:
Unfortunately DOF_Ents is a global table, so addons could be adding things into it.

I also see no reason to change this in the first place as it is not performance intensive code.